docs/02-install.md

```diff
@@ -1,4 +1,4 @@
-# Install
+# Basic/Default Installations
```

Comment:
This is going to make an awkward anchor for linking ;).  Can we just use "Basic Installations"?

Reply:
Sure.